src/pydap/__init__.py

```diff
@@ -3,3 +3,5 @@
 '''
 
 __import__('pkg_resources').declare_namespace(__name__)
+
+__version__ = __import__('pkg_resources').get_distribution(__name__).version
```

Comment:
I'm not a fan of `pkg_resources`, it is slow and hackish.
Could we try something like versioneer? Or maybe define the version in `__init__.py` and parse it in `setup.py`?